authutils/dbgap.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Defines functionality to check case existence in dbGaP.
+"""
+
+import logging
+import requests
+import re
+
+import xmltodict
+from cdislogging import get_logger
+from gdcdatamodel.models import Project
+from xml.parsers.expat import ExpatError
+from cdiserrors import (
+    InternalError,
+    UserError,
+)
+
+PHSID_REGEX = re.compile('(phs\d+.v)(\d)(.*)')
+
+COMPLETE_STATE = ['released', 'completed_by_gpa']
+
+class dbGaPXReferencer(object):
+
+    #: The url from which to pull telemetry reports for project with given
+    #: accession number (to be formatted in)
+    DEFAULT_URL = ('http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/'
```

Comment:
Feels like that should probably https://

Reply:
cc: @philloooo

Reply:
yeah https, this url just redirects to https

Reply:
sgtm